crates/sequencing/papyrus_consensus/src/config.rs

```diff
@@ -70,19 +74,25 @@ impl SerializeConfig for ConsensusConfig {
             ),
         ]);
         config.extend(append_sub_config_name(self.timeouts.dump(), "timeouts"));
+        config.extend(append_sub_config_name(self.network_config.dump(), "network_config"));
         config
     }
 }
 
 impl Default for ConsensusConfig {
     fn default() -> Self {
+        let mut network_config = NetworkConfig::default();
+        // TODO(Dan/Shahak): consider something nicer, maybe a builder?
```

Comment:
How did you choose these ports?
We have a utility for selecting a free port `find_free_port`.
If you prefer the constants, should we move these to the top of the file instead of inline magic numbers?